<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverrideSlaveDialog</class>
 <widget class="QWidget" name="OverrideSlaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Overriding a Slave means taking the place of that Slave.&lt;/p&gt;&lt;p&gt;* You will be able to modify the overriden slave's shots.&lt;/p&gt;&lt;p&gt;* Any delegated shot you have already claimed will remain unchanged.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;WARNING&lt;/span&gt;: This option should be used to create new versions of a Slave and not to take their delegated Shots. The previous Slave file will still be able to modify the Shots so there might be conflicts, proceed with extreme caution!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="slavesPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="slavesHeader">
        <property name="text">
         <string>Slaves</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="slavesList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="ownedShotsPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="ownedShotsHeader">
        <property name="text">
         <string>Owned Shots</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="ownedShotsList">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lowerButtonsLayout">
     <item>
      <spacer name="lowerButtonsLayoutSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="overrideButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Override</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
